apps/web/lib/components/kanban-card.tsx

```diff
@@ -133,62 +142,84 @@ export default function Item(props: any) {
 			data-is-dragging={isDragging}
 			data-testid={item.id}
 			data-index={index}
-			aria-label={`${item.status.name} ${item.content}`}
+			aria-label={item.label}
 		>
-			<div className="grid grid-cols-4 w-full justify-between border-b border-b-gray-200 pb-4">
-				<div className="col-span-3 flex flex-col gap-5 grow w-full">
-					{item.tags && <TagList tags={item.tags} />}
-
-					<div className="flex flex-row flex-wrap text-wrap items-center text-sm not-italic font-semibold">
```

Comment:
@Anishali2 why you removed Issue Status, link etc? i.e. all those lines!? Or you moved them somewhere?

Reply:
Noting is removed I just aligned the css a little bit because there is so much unused css there 
So there are components that changes some position that's why it is changed